CMakeLists.txt.in

```diff
@@ -154,22 +169,104 @@ ExternalProject_Add(gnutls_ext
   INSTALL_COMMAND ${make_command} install
   DEPENDS gmp_ext nettle_ext)
 
+# in cmake >= 3.7
+#   use SOURCE_SUBDIR "build/cmake"
+#   instead of CONFIGURE_COMMAND used below
 ExternalProject_Add(lz4_ext
   URL https://github.com/lz4/lz4/archive/v1.8.0.tar.gz
   URL_MD5 6247bf0e955899969d1600ff34baed6b
   INSTALL_DIR @SMF_DEPS_INSTALL_DIR@
-  BUILD_IN_SOURCE ON
-  CONFIGURE_COMMAND ""
+  CONFIGURE_COMMAND
+    ${CMAKE_COMMAND}
+    -DCMAKE_BUILD_TYPE=@CMAKE_BUILD_TYPE@
+    -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
+    <SOURCE_DIR>/contrib/cmake_unofficial
+  BUILD_COMMAND ${make_command})
+
+ExternalProject_Add(numactl_ext
+  URL https://github.com/numactl/numactl/releases/download/v2.0.12/numactl-2.0.12.tar.gz
+  URL_MD5 2ba9777d78bfd7d408a387e53bc33ebc
+  INSTALL_DIR @SMF_DEPS_INSTALL_DIR@
+  CONFIGURE_COMMAND
+    <SOURCE_DIR>/configure
+    --prefix=<INSTALL_DIR>
+    --srcdir=<SOURCE_DIR>
   BUILD_COMMAND ""
-  INSTALL_COMMAND
-    ${make_command} PREFIX=<INSTALL_DIR> install)
+  INSTALL_COMMAND ${make_command} install)
+
+ExternalProject_Add(libpciaccess_ext
```

Comment:
oh wow awesome. we should remove this then from the `dnf` install and `apt` install